extensions/amp-facebook/0.1/amp-facebook.js

```diff
@@ -41,11 +41,10 @@ class AmpFacebook extends AMP.BaseElement {
     const iframe = getIframe(this.element.ownerDocument.defaultView,
         this.element, 'facebook');
     this.applyFillContent(iframe);
+    const amp = this.element;
     // Triggered by context.updateDimensions() inside the iframe.
     listenFor(iframe, 'embed-size', data => {
-      iframe.height = data.height;
-      iframe.width = data.width;
-      const amp = iframe.parentElement;
+      // TODO REVIEW
```

Comment:
Yeah we can drop that, though I don't think this would cause the actual height and width change cause we're only updating the attributes not the actual style width and height. 

But yes, let's drop them, updating them here seems pretty useless but I don't think it'll cause an immediate resizing.